content/de/ecosystem/contribution/contribution.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Contributions
+description: Wie kann man sich an der Entwicklung von DigiWF beteiligen?
+category: 'Contributions'
+categoryIcon: 'mdi-gift-open-outline'
+position: 1
+---
+
+## Dokumentation
+
+### Dokumentation von Integrationsartefakten
+
+Die Dokumentation von (offiziellen) Integrations Artefakten soll einerseits in dem jeweiligen Repository erfolgen. 
+Andererseits hier in der offiziellen Gesamtdokumentation. Um nicht doppelt Inhalte pflegen zu müssen gelten folgende 
+Regeln:
+
+1. Die Dokumentation erfolgt ausschließlich in englischer Sprache. Auch in andere Übersetzungen (wie beispielsweise 
+   Deutsch) wird die englische Dokumentation eingebunden. 
+2. Die Dokumentation liegt im jeweiligen Repo im Ordner `docs`.
+3. Die Abschnitte der Dokumentation wird in einzelne Abschnitte aufgeteilt. Diese werden dann in das `readme.md` im 
```

Comment:
=> "Die Dokumentation erfolgt in einzelnen Abschnitten." vielleicht besser. Doppeltes "Abschnitte" wird somit vermieden

Reply:
Es müsste Dokumente heißen. Hab ich angepasst.